mozci/push.py

```diff
@@ -980,9 +995,20 @@ def find(runnable_type: str) -> Optional[str]:
 
         return find("label") or find("config_group")
 
+    def _check_build_task_regression(self, task: Task):
```

Comment:
```suggestion
    def _should_retrigger_build(self, task: Task):
```

Reply:
I tried to separate the logic for "task is a build failure" (so I renamed this to `_is_build_failure(self, task)`) and rules for retriggering (in the regression command).